src/components/DetailItems/DetailItem.tsx

```diff
@@ -196,6 +204,7 @@ export const DetailsItem: FC<DetailsItemProps> = ({
   canEdit,
   content,
   crumbs,
+  'data-testid': dataTestId,
```

Comment:
I think we decided on making these props called `testId` when we have to define them on our own.